os/arch/arm/src/armv7-m/up_mpu.c

```diff
@@ -377,3 +377,35 @@ void up_mpu_set_register(uint32_t *mpu_regs)
 		putreg32(mpu_regs[REG_RASR], MPU_RASR);
 	}
 }
+
+/****************************************************************************
+ * Name: up_mpu_disable_region
+ *
+ * Description:
+ *   Update tcb's mpu register values to disable a mpu region.
+ *   MPU region to be disabled is identified using tcb's mpu registers passed.
+ *   So, if the current MPU h/w settings for this MPU region match to those
+ *   of registers passed, set these new register values to mpu h/w as well.
+ *
+ ****************************************************************************/
+void up_mpu_disable_region(uint32_t *mpu_regs)
```

Comment:
Matched API for enabling mpu region is `up_mpu_set_region`. How about changing this API name to unset or changing `up_mpu_set_region` to `up_mpu_enable_region`?

Reply:
I don't think this new API completely matches as the opposite of `up_mpu_set_register`.
This new API is about clearing the Enable bit for MPU region. I believe a matching API for this would be one that just sets this Enable bit.
But` up_mpu_set_register` is about setting all mpu register settings to mpu h/w. The matching API I think would be one that always clears/unsets all of the settings for a region in mpu h/w registers, which this new API does not.
I feel the current naming is fine, especially the way i am using it ( I am calling `up_mpu_set_register` from inside this new API under certain conditions. It will be confusing if I rename and call `up_mpu_enable_region` from inside `up_mpu_disable_region`).

Reply:
Okay. I understood. Let's keep it on then.